src/components/CityColumnChart/CityColumnChart.js

```diff
@@ -153,22 +168,29 @@ const CityBarChart = (props) => {
         />
       </div>
 
-      <AbstractWidget config={config} />
-
       {isStacked && (
         <div style={{ marginLeft }}>
           <Legend
             // Legend is our own HTML so we can style and position it with CSS
             className={style.legendWrapper}
+            title={shortDescription}
             series={series}
           />
         </div>
       )}
+
+      {(!isStacked && narrowViewport) && (
```

Comment:
If this should be conditional on screen size, the logic should be in CSS, since all the code here only runs when the page loads. E.g. it won't run when a user rotates a device.